.github/workflows/enable-pack-build-and-test.yaml

```diff
@@ -59,10 +59,14 @@ jobs:
           fetch-depth: 1
 
       # TODO: Make sure this actually constitutes "repository activity"
-      - name: Set this Workflow to active (prevent 60d disabling)
+      - name: Conditionally set this Workflow to active (attempt to prevent 60d disabling of itself)
```

Comment:
Hrm, this might only restore this workflow's state (if it's disabled) when triggered by `on.workflow_dispatch`...

Reply:
workflow_dispatch is still a good escape hatch if all things automated fail for whatever set of reasons (like we're still figuring out what set of automation is most reliable).